tests/take_screenshots.spec.ts

```diff
@@ -81,11 +81,9 @@ test.describe("The Installer", () => {
       // create storage page screenshot
       await page.locator("a[href='#/storage']").click();
       await expect(page.getByRole("heading", { name: "Settings" })).toBeVisible();
-      // skip this in GitHub CI, there is no suitable disk for installation
-      if (process.env.GITHUB_ACTIONS !== "true") {
-        await expect(page.getByText("Installation device")).toBeVisible({timeout: minute / 2});
-      }
+      await expect(page.getByText("Installation device")).toBeVisible({timeout: minute / 2});
```

Comment:
please, leave the ci code even if does nothing, so we minimize the delta, because this change could be push to agama repo as well, but we can focus for the moment in our own problems :) and then help them when we are a bit mature with playwright I guess.
another reason is that we should have our own test to install and not use `take_screenshots` but that will come when we have severals POMs build.

Reply:
Ok, updated.

Reply:
but then do not copy the line, just remove the condition, wdyt?

Reply:
sorry my bad, that was exactly what you did :(

Reply:
Updated. I remembered tested such code but failed, it maybe for the updated e2e path was wrong configured.